manifests/deployment.yml

```diff
@@ -40,10 +82,7 @@ spec:
             - name: NODE_ENV
               value: stage
             - name: DB_HOST
-              valueFrom:
-                secretKeyRef:
-                  name: semesterly-secrets
-                  key: DB_HOST
+              value: localhost

```

Comment:
I think we want to set this to name of the container = pgbouncer, does that work?

Reply:
This did not work.  When attempted, the container was unable to resolve pgbouncer for the database connection.